doc/guides/dts/index.rst

```diff
@@ -301,7 +301,7 @@ the board's folder and add the extra flags inside.
 Generated macros
 ================
 
-Take the DTS node below as an example.
+Take this devicetree node as an example:
```

Comment:
Thanks for using the correct form. It is indeed, according to the spec `devicetree`, and not `Devicetree`, `Device Tree` or `Device tree`